TestCases/species_transport/venturi_primitive_3species/species2_primitiveVenturi_mixingmodel.cfg

```diff
@@ -0,0 +1,135 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Species mixing with 2 species, i.e. 1 transport equations  %
+% Author: T. Kattmann                                                          %
+% Institution: Bosch Thermotechniek B.V.                                       %
+% Date: 2021/10/14                                                             %
+% File Version 7.3.1 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+SOLVER= INC_RANS
+KIND_TURB_MODEL= SST
+%
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
+%
+INC_DENSITY_MODEL= VARIABLE
+INC_DENSITY_INIT= 1.1766
+%
+INC_VELOCITY_INIT= ( 1.00, 0.0, 0.0 )
+%
+INC_ENERGY_EQUATION= YES
+INC_TEMPERATURE_INIT= 300.0
+%
+INC_NONDIM= DIMENSIONAL
+%
+% -------------------- FLUID PROPERTIES ------------------------------------- %
+%
+FLUID_MODEL= FLUID_MIXTURE
+%
+MOLECULAR_WEIGHT= 28.96, 16.043
```

Comment:
We need some comments here and in config_template regarding how this works, i.e. the "N+1" requirement and how they should be ordered.

Reply:
ok, I will add them. Thank you so much for the feedback